### ISSUE TITLE
discord menu entry

### ISSUE THREAD

**Issue by mahnmut** [anders-larsson/gentoo-overlay#13]
Hi,

Thanks for discord ebuild but please can you add a menu entry.

Just need to add : 
`inherit eutils`

and

```
make_desktop_entry discord Discord \
"/opt/discord/discord.png" \
Network
```

in src_install()

Best regards

**Comment by anders-larsson** [comment]
Feel free to create a pull request :)

**Comment by mahnmut** [comment]
Hope I made it in the right way. I usually use only git bases.

**Comment by anders-larsson** [comment]
Solved by pull request #15.
